ExcelMapper/ColumnInfo.cs

```diff
@@ -105,7 +116,21 @@ public bool IsSubType
         /// <value>
         /// The cell setter.
         /// </value>
-        public Action<ICell, object> SetCell { get; set; }
+        public Action<ICell, object> SetCell
+        {
+            get
+            {
+                if (_setCell != null)
+                {
+                    return _setCell;
+                }
+                else
+                {
+                    return GenerateCellSetter();
```

Comment:
Isn't this called every time a cell is set? Why is this no longer precomputed?

Reply:
Can you elaborate a little?

Reply:
When setcellusing is configured, the user-defined action must be called regardless of whether the type is changed. If not, the built-in generatecellsetter can be called.

Reply:
The function of the above code is to set the background color to red for empty cells through SetCellUsing.

The value of the "Number" member of the first record is null, and the value of the "Number" member of the second record is 60. If its type changes, it will trigger the "ChangeSetterType" call. In the "ChangeSetterType" method, "SetCell = GenerateCellSetter();" will cause the SetCellUsing configured by the user to be discarded. When the third record is processed, SetCellUsing will not be called, which does not match the expected result. Only the background color of the first record is red.

The correct result should be that the background of the first and third records are red, because their "Number" member value is null.